src/password.h

```diff
@@ -0,0 +1,10 @@
+#ifndef PASSWORD_H
+#define PASSWORD_H
+
+#include <cstring>
```

Comment:
`std::string` is in `<string>`, not `<cstring>`. `<cstring>` is [the one](http://en.cppreference.com/w/cpp/header/cstring) that contains `strcmp`, `strcpy` and other C functions that deal with C strings (the `char*` kind).